docs/content/preview/migrate/known-issues/mysql.md

```diff
@@ -267,28 +267,29 @@ CREATE OR REPLACE VIEW v1 AS SELECT foo(bar.id::int) AS p_name FROM bar;
 An example schema on the source database is as follows:
 
 ```sql
-/* procedure definition */
+/* function definition */
 delimiter //
-CREATE PROCEDURE foo(p_id int)
-deterministic
+CREATE FUNCTION func (p_id int)
+RETURNS VARCHAR(20)
+READS SQL DATA
   BEGIN
     DROP TEMPORARY TABLE IF EXISTS temp;
     CREATE TEMPORARY TABLE temp(id int, name text);
     INSERT INTO temp(id,name) SELECT id,p_name FROM bar WHERE p_id=id;
     SELECT name FROM temp;
```

Comment:
Please put the select statement in a return clause.
`return (SELECT name FROM temp);`